.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "prettier.endOfLine": "auto",
+    "files.eol": "\n"
```

Comment:
possible to add this file into .gitignore? Think we typically dont commit this this. It's quite specific to your machine in this case

Reply:
Done